pyfahviewer/static/main.js

```diff
@@ -1,138 +1,124 @@
 (function() {
-  var leaderboard_template = null;
-  var slot_template = null;
-  var fetching = false;
+  const host = location.protocol+'//'+location.hostname+(location.port ? ':'+location.port: '');
```

Comment:
```suggestion
  const host = `${location.protocol}//${location.hostname}` + (location.port ? ':' + location.port : '');
```
I am upset I didn't do this initially. That's what I get for copying and pasting without thinking.

Reply:
There are some other places you can also do string interpolation. Do you want me to update those as well?